docs/data.rst

```diff
@@ -1,8 +1,11 @@
-Input Specifications **(IN PROGRESS)**
-======================================
+Data Specifications
+===================
 
-Directory Setup
----------------
+Directory Setup Recommended
+---------------------------
+
+Developing a custom directory setup is currently in progress, but for now we recommend the
+following directories and names to properly execute ``bmorph``.
```

Comment:
It can be confusing to users when speculation about future development is in the docs. Also, this still isn't a recommendation - just tell the users that they need to have this directory structure to use `bmorph`.